### PR TITLE
Minor change to getting the SystemType reference

### PR DESCRIPTION
**Pull request by davidkline-ms**
This works around Type.Assembly not being available when building with the .NET backend.

This is only one part of fixing getting the compile to work (there are some other features not implemented in the backend that compiles to UWP), but it is a start.

### CODE REVIEW

File: Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs [hunk 1]
```diff
@@ -19,7 +19,14 @@ public sealed class SystemType : ISerializationCallbackReceiver
...
         public static string GetReference(Type type)
         {
-            return type != null ? $"{type.FullName}, {type.Assembly.GetName().Name}" : string.Empty;
+            if (type == null)
+            {
+                return string.Empty;
+            }
+
+            string[] qualifiedNameComponents = type.AssemblyQualifiedName.Split(new char[] { ',' });
+            Debug.Assert(qualifiedNameComponents.Length >= 2);
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs]: Is there a PR for the main vNEXT dev branch I can approve?

**Comment by StephenHodgson** [comment]
@davidkline-ms fixed in https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2231

**Comment by SimonDarksideJ** [comment]
Agreed, this change should be against dev, not the devices PR as it's not related.

File: Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs]: I can spin one up :)

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs]: already fixed.

Comment (davidkline-ms) [reply Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs]: yeah, but I like how this implementation is the same regardless of the backend selected

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs]: yeah but it breaks the serialization in the meta file and corrupts our references

**Pull request closed without merging**